src/plugins/data/public/ui/dataview_picker/change_dataview.tsx

```diff
@@ -0,0 +1,221 @@
+    setIsTourDismissed(true);
+    setIsTourOpen(false);
+  };
+
+  // Create a reusable id to ensure search input is the first focused item in the popover even though it's not the first item
+  const searchListInputId = useGeneratedHtmlId({ prefix: 'dataviewPickerListSearchInput' });
+
+  useEffect(() => {
+    const fetchDataViews = async () => {
+      const dataViewsRefs = await data.dataViews.getIdsWithTitle();
+      setDataViewsList(dataViewsRefs);
+    };
+    fetchDataViews();
+  }, [data, currentDataViewId]);
+
+  const createTrigger = function () {
+    const { label, title, 'data-test-subj': dataTestSubj, ...rest } = trigger;
+    return (
+      <EuiButton
+        css={css`
+          max-width: ${POPOVER_CONTENT_WIDTH}px;
+        `}
+        data-test-subj={dataTestSubj}
+        onClick={() => {
+          setPopoverIsOpen(!isPopoverOpen);
+          setIsTourOpen(false);
+          // onTourDismiss(); TODO: Decide if opening the menu should also dismiss the tour
+        }}
+        color={isMissingCurrent ? 'danger' : 'primary'}
+        iconSide="right"
+        iconType="arrowDown"
+        title={title}
+        {...rest}
+      >
+        {label}
+      </EuiButton>
+    );
+  };
+
+  return (
+    <EuiTourStep
+      content={
+        <EuiText style={{ maxWidth: 320 }}>
+          <p>
+            The data view selector has moved! You&apos;ll find all related data view options in this
+            new menu.
+          </p>
+        </EuiText>
+      }
+      isStepOpen={isTourOpen}
+      onFinish={onTourDismiss}
+      step={1}
+      stepsTotal={1}
+      title={
+        <>
+          <EuiIcon type="bell" size="s" /> &nbsp; Hey there!
+        </>
+      }
+      footerAction={
+        <EuiLink data-test-subj="dataViewPickerTourLink" onClick={onTourDismiss}>
+          Got it!
+        </EuiLink>
+      }
+      repositionOnScroll
+    >
+      <EuiPopover
+        panelClassName="changeDataViewPopover"
+        button={createTrigger()}
+        panelProps={{
+          ['data-test-subj']: 'changeDataViewPopover',
+        }}
+        isOpen={isPopoverOpen}
+        closePopover={() => setPopoverIsOpen(false)}
+        panelPaddingSize="none"
+        initialFocus={`#${searchListInputId}`}
+        buffer={8}
+      >
+        <div style={{ width: POPOVER_CONTENT_WIDTH }}>
+          {onAddField && (
+            <>
+              <EuiContextMenuPanel
+                size="s"
+                items={[
+                  <EuiContextMenuItem
+                    key="add"
+                    icon="indexOpen"
+                    data-test-subj="indexPattern-add-field"
+                    onClick={() => {
+                      setPopoverIsOpen(false);
+                      onAddField();
+                    }}
+                  >
+                    {i18n.translate('data.query.queryBar.indexPattern.addFieldButton', {
+                      defaultMessage: 'Add a field to this data view...',
```

Comment:
We don't typically  include ... after menu items

```suggestion
                      defaultMessage: 'Add a field to this data view',
```

Reply:
Hey @gchaps , so about this. What I'm trying to establish here is a pattern of indicating when menu items will open up a modal/flyout. I've seen this pattern in other places like Google docs that are heavy on menus. It basically indicates that the menu item will not **perform** the action but bring up a longer dialog. I know it's a bit confusing since we also truncate a lot with ellipses. Do you know of any other pattern we can start utilizing?

Reply:
My thought is that using the right verb is enough to suggest what is going to happen from the menu (like opening a popup or a wizard or something outside of the menu itself).  Seeing a menu with a lot of items followed by ... makes the menu harder to read and scan.

The ellipses is the best pattern to indicate when a menu items opens a modal/flyout. It's commonly used by other companies like Microsoft, Apple, and Google to indicate that more input is required. 

We haven't used the ellipsis on menu items in the rest of the UI.  If we start using them here, I think we should review all other menus as well. For example, the Options menu on a dashboard panel has two items that open a flyout or a modal.

Reply:
> We haven't used the ellipsis on menu items in the rest of the UI. If we start using them here, I think we should review all other menus as well.

Yep, exactly. I would like to start a recognizable pattern.

> using the right verb is enough to suggest what is going to happen from the menu

While I disagree generally with this statement, I don't have a strong enough reasoning in this particular menu. But I'd definitely like to start a general discussion around menu items. I'll think about starting a doc, but for now I"ll remove them.

Reply:
+1 to starting a discussion around menu items and making them consistent.